bin/apisix

```diff
@@ -603,6 +631,18 @@ local function init()
         sys_conf["worker_processes"] = "auto"
     end
 
+    local dns_addrs = system_dns_resolver()
+    if dns_addrs then
```

Comment:
I think this logic is not right. If `bin / apisix` fails to read /etc/resolv.conf, then the `dns_resolver` configured in `config.yaml` will also fail.

Reply:
Has been changed to: 

```
        local dns_addrs, err = local_dns_resolver("/etc/resolv.conf")
        if not dns_addrs then
            error("failed to import local DNS: " .. err)
        end
```